Makefile

```diff
@@ -9,7 +9,7 @@ cidocker = docker run ${ciargs} ${dockerargs} golang:1.4
 
 all: ${build_image}.created build check integration-tests
```

Comment:
No I want user to have control on when cleaning.

I want to be able to run 
- `sudo make integration-tests` multiple times, where each run uses the same dnet binary
- `sudo make integration-tests clean` where I compose with clean